cibuildwheel/macos.py

```diff
@@ -19,6 +19,8 @@ def get_python_configurations(build_selector):
         PythonConfiguration(version='3.6', identifier='cp36-macosx_10_6_intel', url='https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.6.pkg'),
         PythonConfiguration(version='3.7', identifier='cp37-macosx_10_6_intel', url='https://www.python.org/ftp/python/3.7.5/python-3.7.5-macosx10.6.pkg'),
         PythonConfiguration(version='3.8', identifier='cp38-macosx_10_9_x86_64', url='https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg'),
+        PythonConfiguration(version='2.7-v7.2.0', identifier='pp272-macosx_10_7_x86_64', url='https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.2.0-osx64.tar.bz2'),
```

Comment:
@mattip What about these version numbers? How does PyPy generally handle/format/use/... versions? The Python vs. PyPy version and how to handle can be confusing. So I'd like to stick close to PyPy conventions.

Reply:
PEP 425 relates to 3 parts to a wheel name: a python tag, an ABI tag, and a platform tag. The python tag relates to the version of the python language itself, the ABI tag relates to the C-level ABI exposed by the implementation, and the platform tag relates to the OS and ancilary library support. On CPython, the python tag and ABI tag track eachother very closely, deviating only for things like malloc models and debugging version. On PyPy the ABI tag reflects the maturity level of the cpyext compatibility layer as well. So PyPy will have a python tag like pypy2.7 or pypy3.6, and a ABI tag that reflects both the python version and the pypy version like pypy27_pp72.

The download names roughly reflect these PEP 425 tags, but not ideally. So the download you added is for python2.7, pypy ABI version 7.20, for osx64.

I don't know if that answers your question

Reply:
I was mostly wondering if the `'2.7-v7.2.0'` makes sense to you. But yes, what you explained clarifies things, indeed. Thanks! :-)

Reply:
What's the thinking behind `pp272` in the identifier here? Is that 'python 2, pypy 7.2.0'?

Reply:
Yes, that used to be the case. I think in the newer versions I've simplified it to `pp27_73` (a shortened version of the new-style PyPy ABI tags)

Reply:
Ah, I see. Thanks. I just posted a little musing on build IDs here - https://github.com/joerick/cibuildwheel/pull/185#discussion_r365382629 , would love your thoughts